instagram/src/Components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import Comments from './Comments'
+import styled from 'styled-components'
+import Likes from './Likes'
+import Input from './Input'
+
+const CommentSectionWrapper = styled.section`
+	max-width: 100%;
+	margin: auto;
+	padding: 2rem 0;
+`
+
+class CommentSection extends Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			comments: props.comments
+		}
+	}
+
+	addComment = newcomment => {
+		const commentAdded = this.state.comments
+		localStorage.setItem('username', 'rhcprocker')
```

Comment:
Later, I would recommend grabbing the username from the localstorage from the login page